.changeset/flat-shirts-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-insights': minor
```

Comment:
I think this is a `patch` release as `tech-insights` is still `<1.0.0`

Reply:
Yeah, was a bit unsure as it is a new feature(kind of) 🤔

Reply:
Yep, new features are 100% minor changes, *unless* the package is in 0.x range. I think it's basically so that you dont have version 38.0.0 as like the first good one after you've made breaking changes as breaking changes are `minor` until you get to 1.0.0 :)

Reply:
Makes sense 😄 Thank you